debug_toolbar/static/debug_toolbar/css/toolbar.css

```diff
@@ -58,7 +56,6 @@
 }
 
 #djDebug[data-theme="dark"] {
-    color-scheme: dark;
```

Comment:
Couldn't this one have stayed?

Reply:
I thought about it and I'm not sure what it even does since we're basically specifying all colors and backgrounds. Yes, it could stay, but then we'd also have to set the opposite on `#djDebug[data-theme="light"]` (I think) 

The bigger point here is that while I do have much experience with CSS generally I don't really know how the `color-scheme` property behaves, and it felt safer to just remove it at this time.

Reply:
Alright that's fair.

Reply:
Btw, sorry for pressing ahead, I wanted to release a better version quickly instead of waiting for more reports.